### ISSUE TITLE
Add multi language support

### ISSUE THREAD

**Issue by ShahanaFarooqui** [ShahanaFarooqui/RTL#196]
Add multi language support

**Comment by bota87** [comment]
If somebody give me a direction I can try to help with this

**Comment by saiy2k** [comment]
Is this a priority? May I take this up?

**Comment by ShahanaFarooqui** [comment]
@saiy2k No, this in not a priority for now. Seems like Chrome can automatically translate applications into other languages. And this feature might be redundant in near future. Have to do some more research before moving forward with this one.

**Comment by ShahanaFarooqui** [comment]
@saiy2k Forgot to add reference tweet. Here is the one https://twitter.com/__B__T__C__/status/1478496225062232064?s=20

**Comment by saubyk** [comment]
Closing this as not planned after a review of where it stands.

In the six years this has been open there has been no demand for it from the user base. The only two comments offering to work on it were from would-be implementers (2020, 2022) rather than from anyone asking for the feature, and there has been no activity at all since January 2022 — where the assessment was already that browser-level translation makes this largely redundant. Nothing since then changes that.

The cost, meanwhile, is a full retrofit rather than finishing something started. There are currently no `i18n` attributes anywhere in the app (205 templates, ~16k lines of markup) and `@angular/localize` is not a dependency — the only thing present is the `extract-i18n` target Angular scaffolds by default. On top of that, RTL commits its build output, and `frontend/` is already ~12 MB; Angular's AOT i18n emits a separate bundle per locale, so each added language would multiply a tracked artifact. A good deal of the vocabulary that matters here — channels, HTLCs, macaroons, sats, keysend, AMP — is not really translated in practice either.

This is the same class of concern as the accessibility work in [RTL-Design#73](https://github.com/Ride-The-Lightning/RTL-Design/issues/73): much cheaper to design into a UI rewrite than to bolt onto the current one. So this is closed as *not planned*, not as *rejected* — if the UI/UX overhaul goes ahead, internationalisation is worth considering as part of it, and if real demand shows up before then this can be reopened on the strength of that.
